### DISCUSSION TITLE
is not valid under any of the given schemas

### DISCUSSION THREAD

**Discussion by bbzcx** [jertel/elastalert2#970] [Q&A] [answered]
I checked everything, but the running rules never executed,In addition to the mailbox parameters, can you help check.
![Uploading 微信图片_20221001114204.png…]()

@greut @dequis @clyfish @louzadod 

[1.txt](https://github.com/jertel/elastalert2/files/9689471/1.txt)

**Comment by jertel** [comment, marked as answer]
There's a typo in the following area:

```
timeframe:
  senconds: 1
```

change to:
```
timeframe:
  seconds: 1
```

Also, you may need to wrap any asterisk/wildcard values in double quotes. Ex: ```index: "pid*"```
Note that 1 second is a very short span, you might need a longer interval.

**Important**: Your post includes what looks like a valid elastic password. If that's true, make sure you change it to ensure your system remains secure.
